statsmodels/graphics/tsaplots.py

```diff
@@ -76,26 +116,14 @@ def plot_acf(x, ax=None, lags=None, alpha=.05, use_vlines=True, unbiased=False,
     else:
         acf_x, confint = acf(x, nlags=nlags, alpha=alpha, fft=fft,
                              unbiased=unbiased)
+    title = 'Autocorrelation'
```

Comment:
another small enhancement: let title be overridden by kwargs
e.g. `title = kwargs.pop('title', 'Autocorrelation')`